README/ReleaseNotes/v636/index.md

```diff
@@ -63,6 +63,42 @@ The following people have contributed to this new version:
   is shared across tasks belonging to different computation graphs.
 - Support for single-threaded snapshotting to RNTuple has been added. This can be enabled through `RSnapshotOptions`. Note that snapshotting from a TTree-based RDataFrame to RNTuple is not yet supported in this release, but will be added in a future version. The current recommended way to convert from TTree to RNTuple is through the [RNTupleImporter](https://root.cern/doc/v636/classROOT_1_1Experimental_1_1RNTupleImporter.html).
 
+### Distributed RDataFrame
+
+The distributed RDataFrame (`ROOT.RDF.Distributed`) is now out of the experimental namespace. We recommend that users switch to using the new version as soon as possible. Note the following changes between the old and the new code:
+
+Using dask:
+```python
+# old version - not recommended: 
+# this will still work in 6.36 but will raise a warning
+df = ROOT.RDF.Experimental.Distributed.Dask.RDataFrame
+
+# new version - recommended: 
+df = ROOT.RDataFrame("mytree", "myfile.root", executor=daskclient
+```
+
+Using spark: 
+```python
+# old version - not recommended: 
+# this will still work in 6.36 but will raise a warning
+df = ROOT.RDF.Experimental.Distributed.Spark.RDataFrame
+
+# new version - recommended: 
+df = ROOT.RDataFrame("mytree", "myfile.root", executor=sparkcontext)
+```
+
+In case of using the old version, with `Experimental` keyword, the warning will be raised. Possibility of using this old version will be completely removed in ROOT 6.38.
```

Comment:
```suggestion
In case of using the old version, with `Experimental` keyword, a warning will be raised. In ROOT 6.38, the `Experimental` name will be removed completely and using it will result in an exception.
```